en/advanced_config/advanced_mc_position_tuning.md

```diff
@@ -1,96 +1,103 @@
-the velocity control loop only is active or both. For the remaining part of this tutorial the term **position-control** represents the position and velocity control loop
-activated and **velocity-control** when just the velocity control loop is in use.
+Higher-level tuning applies to any parameter that has an effect on the desired position/velocity setpoint, in contrast to [lower-level tuning](pid_tuning_guide_multicopter.md) that influences the tracking.
 
-## Precondition
-Before doing any higher-level related control tuning, please follow first the instruction [PID-tuning](https://docs.px4.io/en/advanced_config/pid_tuning_guide_multicopter.html).
-It is not recommended to use the advanced position control tuning parameters as a fix for bad tracking or vibration.
+The set of higher-level tuning parameters can be split into two sections: tuning parameters for [Manual/Stabilized](#manual_mode) mode and tuning parameters for [Mission](#mission_mode) mode. Some parameters will have an effect on both modes. 
 
-## Manual Position Control Mode
-In manual control mode the stick inputs are mapped either to **position-control** or **velocity-control**. Position-control is active when the stick inputs are within 
-the deadzone [MPC_HOLD_DZ](../advanced_config/parameter_reference.md#MPC_HOLD_DZ) and velocity control otherwise. All the parameters below are tuning parameters and cannot
-be mapped directly to the physical quantity. 
+> **Note** The position controller ([diagram here](https://dev.px4.io/en/flight_stack/controller_diagrams.html#multicopter-position-controller)) consists of an outer **P** position-control loop and an inner **PID** velocity-control loop. 
+Depending on the control (flight) mode either both loops are active or just the velocity control loop. 
+For the remainder of this topic the term **position-control** represents the case where both loops are active while **velocity-control** refers to the case when only the velocity control loop is in use.
+
+## Manual/Stabilized Mode {#manual_mode}
+
+In [Manual/Stabilized](../flight_modes/manual_stabilized.md) mode the stick inputs are mapped either to **position-control** or **velocity-control**.
+ 
+Position-control is active when the stick inputs are within the deadzone [MPC_HOLD_DZ](../advanced_config/parameter_reference.md#MPC_HOLD_DZ), and velocity-control otherwise. All the parameters below are tuning parameters and cannot be mapped directly to the physical quantity.
 
 #### MPC_ACC_HOR_MAX
 
-This parameter is used for position-control in the horizontal direction, where the vehicle is supposed to stay at the current location. The limit for the rate of change of the velocity setpoint is defined by 
- `MPC_ACC_HOR_MAX`. This parameter should be set larger than any of the other acceleration related parameters in the horizontal direction.
+This parameter is used for position-control in the horizontal direction, where the vehicle is supposed to stay at the current location. 
```

Comment:
@Stifael So now I'm confused, because this does sound like you mean "position control to keep location fixed in 3d co-ordinate system". Perhaps it is the "Manual/Stabilized" mode that I have wrong - do you actually mean "any mode that has a remote control component"?